src/main/daml/Daml/Finance/Instrument/Swap/Fpml/Util.daml

```diff
@@ -83,13 +84,49 @@ adjustDateAccordingToBusinessDayAdjustments unadjustedDate businessDayAdjustment
     NoAdjustment -> pure $ unadjustedDate
     _ -> getCalendarsAndAdjust unadjustedDate businessDayAdjustments issuer calendarDataAgency
 
+-- | Create the claims for fixed rate payments.
+-- This function supports amortizing notionals.
+-- It will be integrated with the floating utility functions soon, since they have more and more code in common.
+createFixRateAmortizingNotionalPaymentClaimsList : SwapStream -> [SchedulePeriod] -> PeriodicSchedule -> Bool -> Decimal -> Bool -> DayCountConventionEnum -> Decimal -> Deliverable -> [Claim Date Decimal Deliverable Observable]
+createFixRateAmortizingNotionalPaymentClaimsList s calculationSchedule periodicSchedule useAdjustedDatesForDcf couponRate issuerPays dayCountConvention notional cashInstrumentCid = do
```

Comment:
You probably don't need the `do` block as you are within a `List` monad (and don't effectively make use of it)

Reply:
This function will be replaced soon.

Reply:
(As part of this PR: https://github.com/digital-asset/daml-finance/issues/286)